management/server/http/server.go

```diff
@@ -19,12 +19,12 @@ type Server struct {
 	config         *s.HttpServerConfig
 	certManager    *autocert.Manager
 	tlsConfig      *tls.Config
-	accountManager *s.AccountManager
+	accountManager *s.DefaultAccountManager
 }
 
 // NewHttpsServer creates a new HTTPs server (with HTTPS support) and a certManager that is responsible for generating and renewing Let's Encrypt certificate
 // The listening address will be :443 no matter what was specified in s.HttpServerConfig.Address
-func NewHttpsServer(config *s.HttpServerConfig, certManager *autocert.Manager, accountManager *s.AccountManager) *Server {
+func NewHttpsServer(config *s.HttpServerConfig, certManager *autocert.Manager, accountManager *s.DefaultAccountManager) *Server {
```

Comment:
I suggest keeping the AccountManager interface here, then we can easily replace it with some other implementation if needed.
It will also be easier to mock in tests
That is the whole purpose of interfaces :)